news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
1- Mantishub has extended license
2- Admin themes are very good fit for bug-tracker app. Ace was top seller back then when this effort started (around May 2014) 
3- We have full source code control. Also the author is reachable on twitter. 

@badfiles None of your questions are either technical or point to anything strange.  Saying "everything is strange" is counter productive. I am happy to answer any question you may have but let's keep the conversation productive.